src/app/jscaip/state/GameStateWithTable.ts

```diff
@@ -25,43 +25,49 @@ export abstract class GameStateWithTable<P extends NonNullable<unknown>> extends
     public getPieceAt(coord: Coord): P {
         Utils.assert(this.isOnBoard(coord),
                      'Accessing coord not on board ' + coord + '.');
+        return this.getUnsafe(coord);
+    }
+
+    public getUnsafe(coord: Coord): P {
```

Comment:
private plutôt que public si possible ?

Reply:
pas possible, utilisé dans les autres qui ont de l'unreachable

Reply:
ce sont tous des `this.getUnsafe`, donc au moins protected.
NEVER write public unless proven otherwise